.ldrelease/config.yml

```diff
@@ -1,7 +1,12 @@
-template:
-  name: go
-  env:
-    LD_RELEASE_GO_IMPORT_PATH: github.com/launchdarkly/go-test-helpers
+version: 2
+
+jobs:
+  - docker:
+      image: golang:1.14-buster
```

Comment:
Minor: as this library is compatible back to Go 1.13, shouldn't it be released with the 1.13?

Reply:
Just to be clear, "released with" in the case of Go isn't a very meaningful term, because we're not publishing binaries or anything like that. The only effect here is that Releaser by default reruns the unit tests prior to a release, and we're already running those in every compatible Go version in CI; the test run by Releaser is more of a last-minute sanity check just to make sure you haven't completely broken the code. But I'm fine with changing it to 1.13 here.